pkg/agent/nodeportlocal/npl_agent_test.go

```diff
@@ -427,6 +496,76 @@ func TestPodAddHostPort(t *testing.T) {
 	assert.False(t, testData.portTable.RuleExists(defaultPodIP, defaultPort))
 }
 
+// TestPodAddHostPort creates a Pod with multiple host ports having same value but different protocol.
+// It is verified that the Pod's NPL annotation is updated with host port, without allocating extra port
+// from NPL pool. No NPL rule is required for this case.
+func TestPodAddMultiProtocol(t *testing.T) {
```

Comment:
The test case name should include HostPort

Reply:
Updated